definitions/ext-datadog_system/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: DATADOG_SYSTEM
+
+synthesis:
```

Comment:
This synthesis rule can go in this file here:
https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/definitions/ext-host/definition.yml

ext-host's current definition has no synthesis rules, so yours will be the first! Same goes for the golden metrics and summary metrics for ex-host - there are currently none, so you can just go ahead and put all of yours in there. 

For an example of an entity type with multiple rules (from multiple providers), you can look at https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/definitions/ext-service/definition.yml#L4 
This has 3 rules, executed in order

Reply:
Provided you change over directory of the metrics and add the synthesis rule to ext-host, i think we're pretty good to merge!

Reply:
Awesome, moved it over to ext-host. For the WIP stuff to add multiple providers, should I throw a ticket into our backlog to come back and update the metrics when it is released?  I went ahead and created tickets on our side to add some tags so we can be ready to implement it. 

Was also going to move some of the metrics over to dashboards as @naxhh suggested, but that will also need to wait until we get some tags implemented on our side in the above mentioned ticket.

Reply:
@AustinNester considering datadog is the first use case for these two entity types, you'll not need to wait for multiple sources - the next PR will!

Moving some of the metrics would make most sense to do it now. Can you get this done today? I can merge your PR once you're done

Reply:
@cianBuckley Sure thing, moved the metrics over to dashboard.json, let me know if it looks correct!